packages/slice-machine/tests/components/UpdateModal.spec.tsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
I like the idea of the test. But I'm not sure to see the value, we are testing that useSelector is working correctly here right ?

This test can only fail if the user don't display the update command or disconnect the text to the selector, this looks a pretty weak protection against regression. I don't think this component need test because there is no logic inside it except reacting to the props which is not very interesting to test.

Reply:
I could add more to this one to check user actions.

At them moment it checks that if given a condition it will display the popup, it's a condition we're unlikely to see during development. 

The main purpose of these tests are to implement some component based tests.
I added a readme that links to best practices, one of which was ignored here as it would involve testing the main App component.